akka-docs/src/test/java/jdocs/actor/io/dns/DnsCompileOnlyDocTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package jdocs.actor.io.dns;
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import akka.io.Dns;
+import akka.io.IO;
+import akka.io.dns.DnsProtocol;
+
+import static akka.pattern.PatternsCS.ask;
+import static akka.pattern.PatternsCS.pipe;
+
+import scala.Option;
+
+
+public class DnsCompileOnlyDocTest {
+    public static void example() {
+        ActorSystem system = ActorSystem.create();
+
+        ActorRef actorRef = null;
+        long timeout = 1000;
+
+        //#resolve
+        Option<Dns.Resolved> initial = Dns.get(system).cache().resolve("google.com", system, actorRef);
+        Option<Dns.Resolved> cached = Dns.get(system).cache().cached("google.com");
+        //#resolve
+
+        {
+            //#actor-api-inet-address
+            final ActorRef dnsManager = Dns.get(system).manager();
+            ask(dnsManager, new Dns.Resolve("google.com"), timeout);
```

Comment:
Just asking and not doing anything with the returned CS seems strange. Capture in a typed result or maybe add some side effect println or something just to show that we get something back?

Reply:
Agreed :)